packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -7,13 +7,13 @@
         v-for="(route, key) in navigationRoutes"
         :key="key"
       >
-        <template #header>
-          <va-sidebar-item :activeColor="activeColor">
+        <template #header="{ value }">
+          <va-sidebar-item :activeColor="activeColor" @keydown.stop>
```

Comment:
Why `@keydown.stop`?

Reply:
otherwise the event would be triggered twice (on parent and on child) and the expand wil be immediately closed right after it's opened